etcdctlv3/README.md

```diff
@@ -42,3 +42,31 @@ Insert '--' for workaround:
 ./etcdctl put -- <key> <value>
 ```
 
+### DEL [options] \<key\> [range_end]
+
+Removes the specified key or range of keys [key, range_end) if `range-end` is given. A key is ignored if it does not exist.
```

Comment:
I don't think "A key is ignored if it does not exist." is necessary, it's a little confusing.